src/Supply/Application/Dto/ImpressionContext.php

```diff
@@ -92,7 +92,7 @@ function (Zone $zone) {
                 return [
                     'keywords' => $this->user['keywords'],
                     'banner_size' => "{$zone->width}x{$zone->height}",
-                    'publisher_id' => 'pid',
+                    'publisher_id' => Zone::fetchPublisherId($zone->id),
```

Comment:
`Eloquent` model should not be passed here. Application layer shouldn't know about Infrastructure (Eloquent) but I see that it was done in this way before so we'll back to the issue later ... In January ...